dev/create-release/create-release.sh

```diff
@@ -237,7 +237,7 @@ if [[ ! "$@" =~ --skip-package ]]; then
   sbt/sbt clean
   cd docs
   # Compile docs with Java 7 to use nicer format
-  JAVA_HOME=$JAVA_7_HOME PRODUCTION=1 jekyll build
+  JAVA_HOME=$JAVA_7_HOME PRODUCTION=1 RELEASE_VERSION=$RELEASE_VERSION jekyll build
```

Comment:
Since we're updating this line anyway, can we change it to:

```
JAVA_HOME="$JAVA_7_HOME" PRODUCTION=1 RELEASE_VERSION="$RELEASE_VERSION" jekyll build
```

This will protect against word splitting, though unlikely here.